### PR TITLE
Strip sourcemap on sources

### PR DESCRIPTION
**Pull request by pYr0x**
strip source maps now while concatenating files.
also add to the sourcemap regex the `multline` parameter `/m`

this is related to: https://github.com/stealjs/steal/issues/803

with this fix, we now strip all sourcemaps on all files that will be concatenated.

### CODE REVIEW

File: lib/buildTypes/minifyJS.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 var UglifyJS = require("uglify-js");
-var removeSourceMapUrl = require("../remove_source_map_url");
```
Comment (matthewp) [inline lib/buildTypes/minifyJS.js]: I get why you added it to concat, but why did you remove it at this step?

Comment (pYr0x) [inline lib/buildTypes/minifyJS.js]: as i saw, all build process goes through the steam API?
and as i saw, the concatenation is always after transpiling? 

https://github.com/stealjs/steal-tools/blob/2ff7f77322e70cbf7b33484387353ddbcaf088ae/lib/build/multi.js#L89
https://github.com/stealjs/steal-tools/blob/2ff7f77322e70cbf7b33484387353ddbcaf088ae/lib/build/multi.js#L94

do i missed something?

https://github.com/stealjs/steal-tools/blob/792ecdba1b4163518bd3ab3012bc748b4141f350/lib/build/transform.js#L110
https://github.com/stealjs/steal-tools/blob/792ecdba1b4163518bd3ab3012bc748b4141f350/lib/build/transform.js#L140

AND

Comment (matthewp) [inline lib/buildTypes/minifyJS.js]: Oh I see now, because you added the multiline flag to the source map regex it will remove *all* source maps from the concatenated string, is that right?

Comment (pYr0x) [inline lib/buildTypes/minifyJS.js]: yes.

i think that uglify will also strip the sourceMap https://github.com/mishoo/UglifyJS2/blob/41a9329409ca0f1273b6ccdcc9770713f44ec55a/test/mocha/input-sourcemaps.js
